src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -33,6 +33,13 @@
     private String expenseType;
     private String cashAdvanceKey;
     private String reportEntryId;
+    private Boolean reportEntryIsPersonalFlag;
```

Comment:
Yes, this column should be enough to denote the transaction as a personal one, as per Sherry's notes from yesterday's meeting. (The value of "Personal/Non Reimbursable" in the other column is not enough, since there could be other types like "Alcoholic Beverages" that are also personal.)